src/events/InteractionHandler.ts

```diff
@@ -64,4 +71,17 @@ export default class CommandHandler extends Event {
 
         command.execute(interaction);
     }
+
+
+    
+    private handleButtonInteractions(interaction: ButtonInteraction): void {
+        // This function only manages peristent buttons
+        // As of the current design, this only includes the buttons from the Verifier command
+        if (interaction.customId != `Verifier|verifyEmailButton` && interaction.customId != `Verifier|verifyCodeButton`) {
+            return;
+        }
+
+        const command: Verifier = this.client.commands.get("verifier") as Verifier;
+        command.handleButtonInteractions(interaction);
+    }
```

Comment:
Then instead of initializing a command (which doesn't make sense in this context), call the new "handler" function you define from the command code